### PR TITLE
[ENH] Univar regression coefficients

### PR DESCRIPTION
**Pull request by mstrazar**
Fixes to Univariate regression: 
1. Widget outputs regression coefficients
2. Fixed preprocessing. It is now correctly applied to data and visible in the line plot.

### CODE REVIEW

File: Orange/regression/linear.py [hunk 1]
```diff
@@ -149,14 +163,3 @@ def __str__(self):
         return 'LinearModel {}'.format(self.skl_model)
 
 
-class PolynomialModel(Model):
-    def __init__(self, model, polyfeatures):
-        self.model = model
-        self.polyfeatures = polyfeatures
```
Comment (kernc) [inline Orange/regression/linear.py]: The alternative to moving large blocks around, in this case, is patching the type afterwards. I.e., somewhere at the bottom:
```py
PolynomialLearner.__returns__ = PolynomialModel
```

File: Orange/widgets/regression/owunivariateregression.py [hunk 2]
```diff
@@ -17,13 +19,19 @@
...
-    name = "Univariate Regression"
+    name        = "Univariate Polynomial Regression"
     description = "Univariate regression with polynomial expansion."
-    icon = "icons/UnivariateRegression.svg"
+    icon        = "icons/UnivariateRegression.svg"
 
-    LEARNER = LinearRegressionLearner
+    inputs = [("Learner", Learner, "set_learner"),
+              ("Preprocessor", Preprocess, "set_preprocessor"),
+              ("Learner", Learner, "set_learner")]
```
Comment (kernc) [inline Orange/widgets/regression/owunivariateregression.py]: `OWBaseLearner` already defined Preprocessor input.

File: Orange/widgets/regression/owunivariateregression.py [hunk 3]
```diff
@@ -17,13 +19,19 @@
...
+
+    outputs = [("Learner", Learner),
+               ("Predictor", LinearModel),
+               ("Coefficients", Table)]
```
Comment (kernc) [inline Orange/widgets/regression/owunivariateregression.py]: `OWBaseLearner` already defines Learner and Predictor outputs. See its metaclass.

**Comment by ajdapretnar** [comment]
@janezd @mstrazar Why is this a WIP? Can somebody please make a to-do list if there are tasks to be addressed? Thank you! :smile:

p.s. (Yes, nazing around on stale PRs. :heart: )

**Comment by janezd** [comment]
We said we must add or change something, so I added [WIP], but I haven't write a note since this was during the meeting (AFAIR). I'll know for the next time. I hope @mstrazar remembers. :(

**Comment by mstrazar** [comment]
Rebased to apply previous commit from @janezd  and @sstanovnik . I do not recall discussing any direct outstanding issues at the moment.

**Pull request merged**